public/app/plugins/panel/graph3/LineStyleEditor.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useMemo } from 'react';
+import { FieldOverrideEditorProps, SelectableValue } from '@grafana/data';
+import { HorizontalGroup, IconButton, LineStyle, RadioButtonGroup, Select } from '@grafana/ui';
+
+type lineFill = 'solid' | 'dash' | 'dot';
+
+const LINE_OPTIONS: Array<SelectableValue<lineFill>> = [
+  {
+    label: 'Solid',
+    value: 'solid',
+  },
+  {
+    label: 'Dash',
+    value: 'dash',
+  },
+  // {
+  //   label: 'Dots',
+  //   value: 'dot',
+  // },
+];
+
+const SPACKING_DASH: Array<SelectableValue<string>> = [
```

Comment:
```suggestion
const spackingDashOptions: Array<SelectableValue<string>> = [
```

Reply:
btw what is spacking? 

![Screenshot from 2021-01-07 17-45-26](https://user-images.githubusercontent.com/10999/103919282-265eb680-5110-11eb-83ee-973e5ef2485b.png)

Reply:
see also:

twerking,
crumpin': https://www.youtube.com/watch?t=111&v=gCnm0RYyXnU&feature=youtu.be